Release/tests/functional/http/client/proxy_tests.cpp

```diff
@@ -148,13 +148,13 @@ SUITE(proxy_tests)
     TEST_FIXTURE(uri_address,
                  http_proxy_with_credentials,
                  "Ignore:Linux",
-                 "Github 53",
+                 "GitHub 53",
```

Comment:
I'm not sure what this is doing, it's possible it can't be applied... happy to drop if it's a problem...